src/bin/stratisd.rs

```diff
@@ -40,28 +44,44 @@ use devicemapper::Device;
 
 use libstratis::engine::{Engine, SimEngine, StratEngine};
 use libstratis::stratis::{StratisError, StratisResult, VERSION};
+use libstratis::stratis::{alarm, buff_log};
 
 const STRATISD_PID_PATH: &str = "/var/run/stratisd.pid";
+const STRATISD_ALARM_SECONDS: u32 = 600;
+const DEFAULT_LOG_HOLD_MINUTES: i64 = 30;
 
 /// If writing a program error to stderr fails, panic.
 fn print_err(err: &StratisError) -> () {
     eprintln!("{}", err);
 }
 
+/// Print the engine state in a formatted way.
+fn print_engine_state(engine: &Engine) {
+    debug!("Engine state: \n{:#?}", engine);
```

Comment:
I think that someday it would be desirable to find a better thing to output for the engine state than the existing debug info, but this is fine for now.